components/org.wso2.carbon.identity.fraud.detection.sift/src/main/java/org/wso2/carbon/identity/fraud/detection/sift/util/Util.java

```diff
@@ -261,17 +262,39 @@ private static String resolvePayloadData(String key, JsAuthenticationContext con
      * @param context Authentication context.
      * @return Hashed user ID.
      */
-    private static String getHashedUserId(JsAuthenticationContext context) {
+    private static String getHashedUserId(JsAuthenticationContext context) throws FrameworkException {
+
+
+        int currentStep = context.getWrapped().getCurrentStep();
```

Comment:
Can we extract the Authentication Context since its being used multiple times for better readability.